java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +183,42 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Removes and returns up to <code>count</code> members with the lowest scores from the sorted set
+     * stored at the specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zpopmin/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param count Specifies the quantity of members to pop.<br>
+     *     If <code>count</code> is higher than the sorted set's cardinality, returns all members and
+     *     their scores, ordered from lowest to highest.
+     * @return A map of the removed members and their scores, ordered from the one with the highest
+     *     score to the one with the lowest.<br>
+     *     If <code>key</code> doesn't exist, it will be treated as an empty sorted set and the
+     *     command returns an empty <code>Map</code>.
+     * @example
+     *     <pre>{@code
+     * Map<String, Double> payload = client.zpopmax("mySortedSet", 2).get();
+     * assert payload.equals(Map.of('member3', 7.5 , 'member2', 8.0)); // Indicates that 'member3' with a score of 7.5 and 'member2' with a score of 8.0 have been removed from the sorted set.
+     * }</pre>
+     */
+    CompletableFuture<Map<String, Double>> zpopmin(String key, long count);
+
+    /**
+     * Removes and returns the member with the lowest score from the sorted set stored at the
+     * specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zpopmin/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @return A map containing the removed member and its corresponding score.<br>
+     *     If <code>key</code> doesn't exist, it will be treated as an empty sorted set and the
+     *     command returns an empty <code>Map</code>.
+     * @example
+     *     <pre>{@code
+     * Map<String, Double> payload = client.zpopmin("mySortedSet").get();
+     * assert payload.equals(Map.of('member1', 5.0)); // Indicates that 'member1' with a score of 5.0 has been removed from the sorted set.
```

Comment:
Same comment as on Zpopmax. I think this example could be made a little bit more meaningful.